### ISSUE TITLE
Windows Command Processor has stopped working on Windows 10.

### ISSUE THREAD

**Issue by dkiestra** [mridgers/clink#273]
Both clink 0.4.2 and the development version make Windows 10 generate a "Windows Command Processor has stopped working" error when closing Explorer (taskkill /im explorer.exe /f) and then restarting Explorer (start explorer) from the Command Prompt.

It seems to work alright until I start using a batch file and try to do other commands after having restarted Explorer.

**Comment by dkiestra** [comment]
I found that Ansicon 1.66 is causing many issues. When I uninstall Ansicon everything works as expected and I never get the "Windows Command Processor has stopped working" error. It seems Ansicon is causing the error.

**Comment by Bluesman74** [comment]
I rip my CDs using an AUTOIT script called REACT2, which suddenly stopped working when the compression tasks launched they would fail with the 'Windows Command Processor has stopped working' error.  However whenever I ran a command prompt manually everything would be OK.

This thread has given me the solution.  I created a new Command Prompt shortcut launching the 32 bit cmd and this error appeared as I had been experiencing.  I then compared the two environments and found that the variable PROCESSOR_ARCHITECTURE becomes x86 from the 32 bit prompt sol the solution is to edit the autorun as follows:

(IF NOT "%PROCESSOR_ARCHITECTURE%"=="x86" if %ANSICON_VER%==^%ANSICON_VER^% "C:\Cucumber\ANSI160\x64\ansicon" -p).

The actual problem must be attempting to run a 64 bit process from a 32 bit prompt.
